modules/fx/arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/FluxKConnection.kt

```diff
@@ -5,9 +5,9 @@ import arrow.fx.KindConnection
 import arrow.fx.typeclasses.ExitCase
 import arrow.fx.typeclasses.MonadDefer
 
-@Deprecated("Cancelation should be done with the cancelable combinator")
+@Deprecated("Cancellation should be done with the cancelable combinator")
```

Comment:
I believe we discussed this and @nomisRev made the whole API cancelable, single l. Or the other way around hahaha in any case, we should make both uses here consistent.

Reply:
Yep, saw the commit somewhere, I referred to [this article ](https://www.grammarly.com/blog/canceled-vs-cancelled/)

> Canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one L), while cancelled (two Ls) is preferred in British English and other dialects. However, there is only one correct spelling of the word cancellation, no matter where you are.

Reply:
`Cancellation should be done with the cancel*l*able combinator` this is what I mean :D

Reply:
But you can indeed spell it `cancelable`, I just fixed wrong usages of `cancellation`

Reply:
Oh, this again :D I care more about it being consistent throughout the APIs than one or the other.

Reply:
Sure, this one is just about being correct, as `cancelation` doesn't exist. As I mentioned the other usages as `cancelable`, etc are kept